TLC59116_Unmanaged.cpp

```diff
@@ -207,6 +206,13 @@ byte TLC59116_Unmanaged::best_iref(byte ma, int Rext) {
         }
       }
     }
+  // Under conditions where Rext is set to properly limit max current (say 1k resistor for 20mA max output)
+  // the above logic may result in a value for D which is greater than 63.   This value will then get
+  // truncated by the reverse_cc() call (which only takes the lower 5 bits), effectively cutting your output
+  // current setting in half.   It would be better to just rail D.   
```

Comment:
what's the ultimate effect then?

Reply:
See lines that follow (218:220).   If D>63 we just set D>63 (effectively railing to max current allowed by Rext).   If we fail to do this, then D gets truncated by reverse_cc(D) and you end up with half the power.